### ISSUE TITLE
Add an option for displaying the warnings emitted by the compiler on the console

### ISSUE THREAD

**Issue by zosrothko** [gradle/gradle-native#226]
Hi

Actually, none of the warninh emitted by the compiler are displayed on the console. This issue is requesting to add an option for displaying the warning on the console.

**Comment by big-guy** [comment]
@zosrothko thanks!  We'd like to improve the way we handle the interleaving/processing of compiler output so that we either 1) don't have the output.txt file at all or 2) display useful information all the time.  We've put this in the "Soon" bucket, so we'll get back to it after we complete other work.

**Comment by zosrothko** [comment]
@big-guy Having a output;txt file is not quite useful as soon as the compiler's messages are displyed on the console. Both on Linux and Window stdout/sdterr can be redirected to a file if needed. Thus I would suggest to not provide the output.txt but display all messages from the compiler/linker on the console.

**Comment by kpenev** [comment]
A presumably very simple step in that direction could be a simple report at the end of a build of the number of warnings encountered overall. At least this way developers with normally clean builds will be alerted to check their output.txt files.

**Comment by lacasseio** [comment]
We should just remove [the operationLogger](https://github.com/gradle/gradle/blob/4c751cb0d64011a22fb0b30061b3057a219ee6aa/subprojects/language-native/src/main/java/org/gradle/language/nativeplatform/tasks/AbstractNativeCompileTask.java#L118) from [everywhere it's being used in native ](https://github.com/gradle/gradle/search?utf8=%E2%9C%93&q=newOperationLogger&type=).

**Comment by lacasseio** [comment]
To solve this issue, we should remove the operationLogger from the various native tasks (I would probably just add a identity BuildOperationLogger that just pass everything to the console but still log to the outputs.txt file (just in case). We can deprecate the `output.txt` file or not later. Then, we would need to ensure no test are broken. We can remove the tests that are expecting the output from being truncated or just reverse them to expect everything.

For the tests that already output warnings (but were hidden), with fix the compiled code or force the no-warning compiler flags so we can. If you do force a no-warning compiler flags, we should gather all the place where this was added and open an issue for some follow-up cleanup.

Basically, this issue should be focused at disabling "pass-through" the entire output of the compiler only. The `outputs.txt` file should still be generated and we shouldn't aim at removing any API on this first change.

I welcome anyone who wants to improve the native plugins. This has being requested time and time again.
